src/atoms/search.ts

```diff
@@ -0,0 +1,3 @@
+import { atom } from 'jotai'
+
+export const searchKeywordAtom = atom<string | undefined>(undefined)
```

Comment:
```suggestion
export const searchKeywordAtom = atom<string>('')
```
빈 문자열로 초기화할 수 있을까요?!

Reply:
[반영 완료](https://github.com/price-offer/offer-fe/pull/241/commits/4dc6b204eaad5c8f3b150fc45139a90e9d72c449#diff-9954859a4a97d13b3edc6d15e762d0e78e8a81338c3f0875efff335964b2d5f6R3)